includes/common.inc

```diff
@@ -72,7 +72,7 @@ function print_warnings() {
 ######################################################################
 
 # Connect to the portsdb and gather miscellaneous information:
-function portsdb_connect() {
+function portsdb_gather_information() {
```

Comment:
You're right that this function no longer takes a username and password (because we've configured PostgreSQL on our new server to use UNIX user accounts for authentication, which no longer require a username and/or password). However, this code does in fact connect to the database by calling `pg_connect("")` in line 80, so I'd say the function is appropriately named.

I agree we can drop the parameters, though.

Reply:
It doesn't return the connection though. It might connection but that is not its goal.

Reply:
Good point, it returns other data as well, so I guess that's fine then.